src/styles/utils/variable.scss

```
@@ -0,0 +1,12 @@
+$gap: 1px;
+$size: 100px;
+$padding: 10px;
+$day-count: 7;
+$day-numder:
+  'mon' 0,
+  'tue' 1,
+  'wed' 2,
+  'thu' 3,
+  'fri' 4,
+  'sat' 5,
+  'sun' 6;
```

If there are problems with the linter, then you can take it all in round brackets
```suggestion
$day-numder: (
  'mon' 0,
  'tue' 1,
  'wed' 2,
  'thu' 3,
  'fri' 4,
  'sat' 5,
  'sun' 6,
);
```